src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -666,8 +666,520 @@ app, err := newrelic.NewApplication(
 )
 ```
 
+Not all possible configuration options may be set via environment variables. The following table lists all of the available configuration functions and their corresponding environment variables. Although any named configuration option may be set by directly assigning a value to the corresponding field in the `Config` structure, we recommend using configuration functions and/or environment variables whenever possible.
+
+Here are some tips for how to use the table:
+
+* If an environment variable is listed in the table, then you may set the corresponding option by setting the named environment variable. You must also include the `ConfigFromEnvironment()` function, which will cause the agent to accept all `NEW_RELIC_*` environment variables.
+* If a configuration function is listed, you can use that function to set the corresponding option instead of using `ConfigFromEnvironment()`. Keep in mind that the configuration functions listed in the program, including `ConfigFromEnvironment()`, are resolved in the order they appear in the code. This means that if you create an environment variable and call the function `ConfigFromEnvironment()`, it will overwrite corresponding configurations you may have set previously using a specific function. Subsequent configuration options after `ConfigFromEnvironment()` will override previous configuration functions and environment variables.
+* See the documentation here and at [the Go documentation site](https://pkg.go.dev/github.com/newrelic/go-agent/v3@v3.20.0/newrelic#ConfigOption) for more information about how to use each function.
+
+
+<table>
```

Comment:
huge table. consider we nest other big sections in this doc, should we nest this in a collapser and say 'If you want to learn more about which ones can and cannot be configured via env variable, see below...'

Reply:
@zuluecho9 I moved the bullet list and table into a collapser.